js/src/dapps/registry/accounts/accounts.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+import Menu from 'material-ui/Menu';
+import MenuItem from 'material-ui/MenuItem';
+
+// import styles from './lookup.css';
+
+const renderAccount = (active) => (account) => (
+  <MenuItem
+    key={ account.address }
+    value={ account.address }
+    primaryText={ account.name }
+    style={ active && active.address === account.address ? { color: 'red' } : {} }
+  />
+);
+
+export default class Accounts extends Component {
+
+  static propTypes = {
+    actions: PropTypes.object.isRequired,
+    all: PropTypes.object.isRequired,
+    selected: PropTypes.object
+  }
+
+  componentDidMount () {
+    // TODO remove this
+    this.props.actions.fetch();
+  }
+
+  render () {
+    const { all, selected } = this.props;
+
+    return (
+      <Menu
+        value={ selected ? selected.address : null }
+        onChange={ this.onAccountSelect }
+      >
+        { Object.values(all).map(renderAccount(selected)) }
```

Comment:
We might need a es-shim: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_objects/Object/values